eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -11,4 +11,4 @@ steps:
       workingDirectory: $(Build.SourcesDirectory)/${{ parameters.Directory }}
       filePath: eng/common/scripts/Verify-Links.ps1 
       arguments: >
-        -urls $(dir -r -i *.md) -rootUrl "file://$(Build.SourcesDirectory)/${{ parameters.Directory }}" -recursive:$false -ignoreLinksFile ${{ parameters.IgnoreLinksFile }}
```

Comment:
Can you split this into multiple lines? With `>` already there you can just split.

```
-urls $(dir -r -i *.md) 
-rootUrl "file://$(Build.SourcesDirectory)/${{ parameters.Directory }}" 
-recursive:$false 
-ignoreLinksFile ${{ parameters.IgnoreLinksFile }}
```